### ISSUE TITLE
Full answer text

### ISSUE THREAD

**Issue by dennmuel** [jakelauer/allbadcards#206]
I don't particularly miss this feature, but I thought I'd note it anyway.

Maybe we could merge the white card into the black card when it's turn to show the answers, e.g.: "I want _ for Christmas." and "a nice gift" would be merged to one (grey?) card saying "I want **a nice gift** for Christmas." (with emphasis on the response text).

**Comment by jakelauer** [comment]
Have had several requests for this and I like the idea. The main issue is figuring out the rules for replacing text. I think it would be better suited to just showing up in the black card rather than a gray card.

First stab at the rules:
```
- Loop through each selected white card
    - Find the first "_" in the black card
    - If there are no "_", append the answer to the end
```

**Comment by qwertpi** [comment]
> Have had several requests for this and I like the idea. The main issue is figuring out the rules for replacing text. I think it would be better suited to just showing up in the black card rather than a gray card.
> 
> First stab at the rules:
> 
> ```
> - Loop through each selected white card
>     - Find the first "_" in the black card
>     - If there are no "_", append the answer to the end
> ```

I agree that black would be better than grey and would love to see this feature implemented

**Comment by dennmuel** [comment]
The grey card thing was just an optional idea to help visualize that black and white cards merged, in case people wouldn't get it. I'm absolutely fine with a black result card. :)

Do you think this should be an optional feature (so one can choose to experience it more like the printed game) or just the new way that ABC behaves?
